docs/tutorial.md

```diff
@@ -17,8 +17,8 @@ The tutorial below uses OpenAI's gpt-4o model, so make sure you have `OPENAI_API
 You can also use other models, keep reading to learn more. Just make sure their keys are properly set.
 
 !!! note
-    An easy way to set environment variables is to use a `.env` file in your project's root directory and use `python-dotenv` to load them.
-    Our [example project](https://github.com/vintasoftware/django-ai-assistant/tree/main/example#readme) uses this approach.
+An easy way to set environment variables is to use a `.env` file in your project's root directory and use `python-dotenv` to load them.
+Our [example project](https://github.com/vintasoftware/django-ai-assistant/tree/main/example#readme) uses this approach.
```

Comment:
Seems to be an automated change. But does it break rendering?

Reply:
Yep, it's broken